package/firmware/omnia-mcu-firmware/Makefile

```
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2023 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-mcu-firmware
+PKG_DISTNAME:=omnia_hw_ctrl
+PKG_VERSION:=3.4
+PKG_RELEASE:=$(AUTORELEASE)
```

Unfortunately, autorelease was deprecated here, though. :-( It was handy feature and I am sad every time when someone forgets to bump the version.

```suggestion
PKG_RELEASE:=1
```

---

ok